src/soundRoadRunner.base64.ts

```diff
@@ -0,0 +1 @@
bGvqiRfXJgUEY6MjItMjpOaICIkJSWm6WtsrKvraWelI+NjYV7bWNdWlhVWFpgYmJkanR3eH2AgoWFiIiLi4iHi5KWl5aSkZCNi4eFg358e3l5fH19e3h4eHl7e3t4dnNydHZ7fHl5eXl+ho2QkZGPjIeHj5CRkIeCfn6BgoKBgIGBgoJ+fX6BgXtuZWJjaGxtdH2DeWJTWHKZqJR7aXaRjJalo62bXlN0yOG5czotZZ+gnIuGdE5Vfbfcxl4tN4C+rZp2ZHZxZ5C5uY01N1iqw4ZERGqVkYufm4hpTm/BzaBGPFiRpYJpb3hkW2iaqoxLPICkrYFaaYCLg4ySlH1VX6O+qIJieZyon42LiHZziKCogV1acpaVhnRqaWp0iIt0Ylp0lJaFbWpygIuRkYh0dICZoJGAc3yKjId7b21udHx8cVpTXXiLg2VaWmmFioFvXVppgpaUgGpugqCqoJGDjZ+oraWclJGZo6SahXh3g5KLcmJaYnN9fG9jYGx7gH50cXiCkZqVkIqRoaiqpJeWoK20sqOUkZqgno93bGdqc3JjSzc1RlNQSTItOkZbYF5aWGd8iI+Hfn6Mn6ilm5GUnqitqJqRj5egn5B8b3F3fHZtX1tjbXZyamdpdoaPkY2PkqO0ubSopK+8vreqnJucoZ+Xint0dnl5bV9QS1NaW1A8Oj9QXV1YUFVdbn6GhoKDkaOqqp+Wl6OqqKGVi4aFg4B4cmhlbG9tYFNTWmdsamVkbHOBioyPjZKfqKqqnJ+vvLy0qJyanJ6ckoN5cXF0c2VVS0lOWF5YSURLVWdsZ2VpdIKNkpSRkZWjr6+qoJyjpaWekIiCgYKCe29jXl5jZGBaWFpfZ25vb3N7g4yQkJSan6itra2qqq2vr6qjnpaUkY2FeXFubGhkYl5bWl1bW1tbYGhtcnJyfYaIjI2NjY2QlJaVjIaDhYeFgHhyc3NxbmlnY2JjZGdkYmJlbXN0c3d9hYuQlJaZnqOoqqilpKWlpKCblI2KiIaAeHNxcnJvbGdlaW1xc3Nzc3d+goOFhYeNkpeXlJGRlJaUlpGLhoGAfnZtZ2JkZWBbWFVaXWJkYmJlbXd+g4aHipCXm5uZl5mbn6CempWSkZGPiIF8d3FvcWxjXVpbYmRkZGdueHyAgoaKkJaanJ+ho6SoqKWhmpeXlpaQh4B7d3RxbWhgW1paWlpaWFhaXWVtc3Z4fIOIkJWXl5qbn6GhoJyZlpSQi4iBe3Z0c3FtaGBeX2JjY2BgYmdtcXR3eXyCipSZmpucn6SoqKikoaCgn5yWjYiFgXx4cWplZGJfXl1aWltdYmVoaGludHh8gIKFipCUlpucnp6cnp6cm5WNi4uIhX14cWpnZGNgX15bXWBjZWVnaW50eHt9g4iNkJWanp+go6WlpaOgn56cmZWPioWCfXhzb2pnZGNiYGBgY2RnaW1xdHh5fYOHio2QkpWXm5ubm5qZlpWUkY+Ig355dnFqaGNfXVtaWFhVWFpfZGdpbXJ3fYKHioyQlpqenp6enJyenpyXkpCPjIiFfnt4d3Rxbm1sbG1ub25vc3Z7gIWGioyRlZeamZmXl5eXlpKPjIuIhoOAfHl2cW1qZ2RjYmJiYmNnam1vc3R4fYKFiIyPkJGUlJWSkIyLioeFgX17eXh3dHNycW9xcW9xb3Jzdnd5e36ChYeLjZCRlJWXl5aUlJKSkIyLh4OAfXt3dHJxcW9vb29vcnN0dnl5fH1+goWGiIiLjI2Nj42NjYyMjIqHg4B9e3h0cW9tbWxpaWdoaGhqbm9ydnh8gIWHiIyPkZKUlpaWlZSSkpCNi4eGgoB8e3h2cnFxb29vb29xcXJ0d3l8foKFiI2QkZSWl5aXl5aWlJGQjIuHg4F9eXRzcW5saWhnZ2dnZWVnaGxucXJ2eX2Ag4eKjI2QkZKSkZCPjYuHhYOAfHh2cnFubGpqbGxsbm9zdnh5fICDhoiMkJKUlpeZm5ubm5ybmpeVlJGNioeFgX17d3NxcW1sampqbGpsbnFyc3d4e3x+gIKDg4WFhYaGhYWDgoGAfn17eHd0c3Fxbm1tbWxtbm5vcXN2eX2Ag4aKjZCRkpSVlZWWlpaWlZSUkpGQj4uIhoaDgX59e3h3dnd2c3N0dnd4eXt9foCBgoOFhoeHh4eGhYOCgX59fHl4d3ZzcnJxcXFvb29xcXJzdHZ3eHt9fYCBg4WGiIqKi4yNjY2Mi4qIh4aGhYOCgoF+fHx7eXl4eHh7fH1+gYOFhoeIiouKi4uLi4qKiIeFhYKAfn17eXh3dnR0c3R0dHZ0dnZ2dnd4eHh5e3x8fX1+fn5+fn5+fn5+fn6AgICAgIGBgYKCg4KDg4ODhYaFhoeHh4eIiIiKioiKiIiIh4aGhYODgoKCgYGAgH5+fn19fXx8e3l7eXh4eHl4eHh3eHh4eXl5e3t5e3t7e3t7e3t7e3t8fHx8fH19fX1+fn5+foCAgICBgoKDgoWFhYaGh4eHiIiIiIiIioiIh4iHh4eHhoaGhYWFg4KCgoKBgYCAfn59fHx7eXl5eHh4eHh4eHh4d3h4eHh4eHd4eHh4eHl5eXt7e3t7fHx9fX59fn5+fn5+foCAgICBgYKCgoKDhYWFhYaGhoaGhoaGhoaHhoeHh4eHh4eHh4eHhoaGhYWDg4KBgYB+fn18fHt7e3l4eHh4d3h4d3d4eHh3eHd4eHh5e3t7e3x8fXx9fn5+foCAgYCBgYGBgYKCgoKCgoKCgYKBgoKCgoKDgoODg4KDg4KDg4WFhYWFhYWFhYOFg4ODg4OCg4KDgoKBgYCAfn19fXx7e3t5eXl5eHh4d3h4d3h4eHh5e3x7fH19fX19fX19fX19fn6AgICAgICBgIGBgYGBgYGCgoGCgoKCgoKCg4ODg4WFhYWGh4eGh4eHh4eHiIeGhoaGhYWDg4KBgIB+fX18eXl5eXh3eHh3dnZ0dHR0dHR2dnd3d3h4eXl7e3x9fX19foCAgICBggAA`;
```

Comment:
Could you add a reference to the Readme (if the art is licensed)?

Reply:
That's actually a problem, I cannot find a proper licence for it. I implemented it for private use, but I think it would be better to remove due to missing licence.

Reply:
ok sure, you could also reuse the existing sound .. there are good ones at https://opengameart.org/